roxctl/analyze/netpol/npgLogger.go

```diff
@@ -0,0 +1,24 @@
+package netpol
+
+import (
+	"github.com/stackrox/rox/roxctl/common/logger"
+)
+
+// npgLogger wraps the common logging interface in a NP-Guard compatible logger.
+// It mutes all warnings and errors as they are being returned explicitly by the library.
+type npgLogger struct {
```

Comment:
This looks familiar already - isn't it a duplication of the generate netpol code?

Reply:
moved to the new pkg` roxctl\common\npguardutils`